packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -588,7 +588,7 @@ export const [
         element,
         DigitalGreenLight,
         DigitalGreenLight,
-        PowerGreen
+        hexToRgbaCssColor(White, 0.6)
```

Comment:
When I look at the anchor in storybook for your PR, the prominent link on the color theme still looks green. Am I missing something?

Reply:
I'm only changing `linkActiveProminentFontColor`, so you'll see the difference only when clicking down on the link.